src/util/DeleteComponent.tsx

```diff
@@ -78,8 +61,7 @@ function DeleteComponent({
     };
     return (
         <div>
-            {!showCannotDeleteDialogModal && renderDeleteDialog()}
-            {showCannotDeleteDialogModal && renderCannotDeleteDialogModal()}
+            {!showDeleteDialogModal && renderDeleteDialog()}
```

Comment:
reverse all the checks & setter values as state name has been changed from `showCannotDeleteDialogModal` to `showDeleteDialogModal`